raspbian/applications/camera.md

```diff
@@ -645,14 +645,13 @@ Outputs debugging/information messages during the program run.
 ```
 	--timeout,	-t		Time before the camera takes picture and shuts down
```

Comment:
As this is part of the `raspivid` documentation, I presume the "takes picture" is wrong too?